packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/Expressions/TupleExpressionTests.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using Microsoft.Generator.CSharp.Expressions;
+using Microsoft.Generator.CSharp.Primitives;
+using Microsoft.Generator.CSharp.Providers;
+using NUnit.Framework;
+using static Microsoft.Generator.CSharp.Snippets.Snippet;
+
+namespace Microsoft.Generator.CSharp.Tests.Expressions
+{
+    internal class TupleExpressionTests
+    {
+        [Test]
+        public void VerifyTupleExpressionWrite()
+        {
+            var tupleExpression = new TupleExpression(Null, Static<CancellationToken>().Property("None"));
+            using CodeWriter writer = new CodeWriter();
+            tupleExpression.Write(writer);
+
+            Assert.AreEqual("(null, global::System.Threading.CancellationToken.None)", writer.ToString(false));
+        }
+
+        [Test]
+        public void VerifyTupleExpressionAssignment()
```

Comment:
nit: could we please also add a testcase showing the variable tuple expression usage with `ref` ?

Reply:
added